samples/wifi/provisioning/implementation_details.rst

```diff
@@ -0,0 +1,268 @@
+Characteristic Name        UUID
+Information                14387801-130c-49e7-b877-2881c89cb258
+Operation Control Point    14387802-130c-49e7-b877-2881c89cb258
+Data Out                   14387803-130c-49e7-b877-2881c89cb258
+========================== ====================================
+
+The characteristic requirements of the Wi-Fi Provisioning Service are shown in table below.
+
++-----------------+-------------+-------------+-------------+-------------+
+| Characteristic  | Requirement | Mandatory   | Optional    | Security    |
+| Name            |             | Properties  | Properties  | Permissions |
++=================+=============+=============+=============+=============+
+| Information     | M           | Read        |             | No Security |
+|                 |             |             |             | required    |
++-----------------+-------------+-------------+-------------+-------------+
+| Operation       | M           | Indicate,   |             | Encryption  |
+| Control         |             | Write       |             | Required    |
+| Point           |             |             |             |             |
++-----------------+-------------+-------------+-------------+-------------+
+| Operation       | M           | Read, Write |             | Encryption  |
+| Control         |             |             |             | Required    |
+| Point           |             |             |             |             |
+| - Client        |             |             |             |             |
+| Characteristic  |             |             |             |             |
+| Configuration   |             |             |             |             |
+| descriptor      |             |             |             |             |
++-----------------+-------------+-------------+-------------+-------------+
+| Data Out        | M           | Notify      |             | Encryption  |
+|                 |             |             |             | Required    |
++-----------------+-------------+-------------+-------------+-------------+
+| Data Out        | M           | Read, Write |             | Encryption  |
+| - Client        |             |             |             | Required    |
+| Characteristic  |             |             |             |             |
+| Configuration   |             |             |             |             |
+| descriptor      |             |             |             |             |
++-----------------+-------------+-------------+-------------+-------------+
+
+Information
+-----------
+
+This characteristic is used to get Info message.
+when read, the Information characteristic gives an encoded Protocol Buffers message in type of Info defined in version.proto.
+It is used by client to determine which version of Wi-Fi provisioning service is running on this server.
+
+Operation Control Point
+-----------------------
+
+This characteristic is used to send Request message and get Response message.
```

Comment:
used to send a request and get a response.

The second sentence repeats the first in unnecessary detail.

Reply:
Reorganized this part.